src/object_ref.jl

```diff
@@ -35,13 +38,58 @@ function has_owner(obj_ref::ObjectRef)
     return !isempty(ray_jll.SerializeAsString(owner_address))
 end
 
+function _register_ownership(obj_ref::ObjectRef, outer_obj_ref::Union{ObjectRef,Nothing})
```

Comment:
assuming the coverage miss here is real, we should flag that this is not used anywhere and link to the relevant issue/comment:
```suggestion
# XXX: this is not currently used pending investigation of how to properly handle ownership
# see https://github.com/beacon-biosignals/Ray.jl/issues/77#issuecomment-1717675779
# and https://github.com/beacon-biosignals/Ray.jl/pull/108
function _register_ownership(obj_ref::ObjectRef, outer_obj_ref::Union{ObjectRef,Nothing})
```